src/dmd/link.d

```diff
@@ -34,6 +35,23 @@ version (Posix) extern (C) int pipe(int*);
 version (Windows) extern (C) int spawnlp(int, const char*, const char*, const char*, const char*);
 version (Windows) extern (C) int spawnl(int, const char*, const char*, const char*, const char*);
 version (Windows) extern (C) int spawnv(int, const char*, const char**);
+version (Posix) extern (C) extern __gshared char** environ;
+version (CRuntime_Glibc)
+{
+    // borrowed from https://github.com/dlang/druntime/commit/51a9962482a668fab00cc866045d76a9c4a39b63
+    // remove and replace with core.sys.posix.spawn once 2.083 is bootstrap
```

Comment:
Using:
```
static if (__VERSION__ >= 2083) import core.sys.posix.spawn;
else
{
 // Decls
}
```

Gives you the best of both worlds

Reply:
done